src/java.base/share/classes/sun/security/jca/ProviderConfig.java

```diff
@@ -39,6 +39,8 @@
 
 import sun.security.util.PropertyExpander;
 
+import openj9.internal.security.RestrictedSecurity;
```

Comment:
The first year in the copyright notice for this file should be 2022 as that is when this work began.